src/code_80091750.c

```diff
@@ -12987,9 +12987,9 @@ void func_800AA2EC(struct_8018D9E0_entry *arg0) {
     case CONFIRM_OK_SELECTION_FROM_BACK_OUT:
     case TIME_TRIALS_DATA_SELECTION_FROM_BACK_OUT:
         if ((arg0->unk4 == 0) && (temp_v0 = D_8018EDF3, (temp_v0 == 1)) && (*(&D_800E86AC - 1 + temp_v0) == 1)) {
-            if (D_800E86F8 != 0) {
+            if (controller_pak_1_state != 0) {
                 sp20 = 0;
-                temp_v0_2 = osPfsFindFile(&D_8018E868, D_800E86F0, D_800E86F4, &D_800F2E64, &D_800F2E74, &D_8018EB84);
+                temp_v0_2 = osPfsFindFile(&gControllerPak1FileHandle, company_code, game_code, &game_name, &ext_code, &gControllerPak1FileNo);
```

Comment:
Could you please rename temp_v0_2 to fileNumber?

I wouldn't normally ask for extra stuff like this, but I think good change.

Reply:
ok